src/main/resources/db/migration/V1__init.sql

```diff
@@ -35,12 +35,24 @@ CREATE TABLE contacts (
     ON DELETE CASCADE
 );
 
+CREATE TABLE approvements (
+  task_id      INT,
+  user_id      INT,
+  mark         ENUM ('GOOD', 'BAD', 'VIEWED') DEFAULT 'VIEWED',
```

Comment:
Nice counting. I believe, that computers (and db) exist to make their work. 
Task can be VIEWED, but have no mark. I think, it should be like that, and i see no arguments to delete VIEWED status, or make one method instead of one.
Еще раз.
Список задач непросмотренных. Кликает оператор на задачу - все, аппровмент есть, она имеет статус просмотренной. Оператор под чаек начинает изучать задачу. Допустим, там не пять тестов и функция фиббоначи, а десять тестов и десять функций. Проходит час. Затем он решает, что задача решена хорошо/плохо. (это может быть вообще через день, а может он просто хочет ее пометить как просмотренную и не проверять). Поэтому необходим статус VIEWED отдельно от GOOD, BAD.
 И поэтому необходимо иметь два метода.